easybuild/easyconfigs/h/HDF5/HDF5-1.8.12-foss-2018b.eb

```diff
@@ -0,0 +1,31 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+name = 'HDF5'
+version = '1.8.12'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': True, 'pic': True, 'usempi': True}
```

Comment:
@sassy-crick Please drop the `'optarch': True, ` (since that's enabled by default)

Reply:
Done. However, it should be noted that it still was in the HDF5-1.8.13-foss-2018b.eb which already made it into the repository.